libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -127,10 +123,14 @@ AC_WPNav::TerrainSource AC_WPNav::get_terrain_source() const
 ///     should be called once before the waypoint controller is used but does not need to be called before subsequent updates to destination
 void AC_WPNav::wp_and_spline_init(float speed_cms)
 {
+    
+    // sanity check parameters
```

Comment:
Let's remove this extra comment at the top, it's pretty obvious what's going on from the other comment immediately below.